force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,7 +167,19 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
+        if (score >= 90) {
+        return 'A';
+        }
+        if (score >= 80) {
```

Comment:
It is fine to do it like this with separate if statements because you are returning, but you can make it clearer by making these `else if` statements, and save a line